cedar-policy-validator/src/cedar_schema/err.rs

```diff
@@ -372,6 +372,9 @@ pub enum ToJsonSchemaError {
     /// Invalid type name
     #[error("this uses a reserved namespace or typename: {}", .0.node)]
     ReservedName(Node<SmolStr>),
+    /// Use reserved JSON schema keywords
+    #[error("this uses a reserved JSON schema keyword: {}", .0.node)]
+    ReservedJsonSchemaKeyword(Node<SmolStr>),
```

Comment:
I think these keywords are also reserved in the Cedar syntax?  So the error message could just say "reserved schema keyword"

Reply:
Done.